src/presentation/ui/page/databases.templ

```diff
@@ -0,0 +1,299 @@
+	@DatabasesIndexLocalState()
+	<div class="flex flex-col" x-data="databases">
+		<div class="mb-6 flex flex-row items-center justify-between">
+			@componentStructural.PageTitle(
+				"Databases", "Manage your database services by adding and deleting databases or users.", "ph-database",
+			)
+		</div>
+		<div
+			id="databases-content"
+			hx-get={ "/databases/?dbType=" + selectedDbTypeSummary.Type.String() }
+			hx-trigger="submit from:form delay:500ms, click from:button#delete-element delay:500ms, click from:button#create-database-user delay:500ms"
+			hx-select="#databases-content"
+			hx-target="#databases-content"
+			hx-swap="outerHTML transition:true"
+			class="card w-full"
+		>
+			<div id="databases-tabs">
+				@DatabasesTabs(selectedDbTypeSummary)
+			</div>
+			@CreateDatabaseModal(selectedDbTypeSummary.Type)
+			@componentStructural.DeleteModal(
+				"isDeleteDatabaseModalOpen", "closeDeleteDatabaseModal()",
+				"deleteDatabaseElement('"+selectedDbTypeSummary.Type.String()+"')",
+				"database.name", "",
+			)
+			@CreateDatabaseUserModal(selectedDbTypeSummary.Type, selectedDbTypeSummary.Databases)
+			@componentStructural.DeleteModal(
+				"isDeleteDatabaseUserModalOpen", "closeDeleteDatabaseUserModal()",
+				"deleteDatabaseUserElement('"+selectedDbTypeSummary.Type.String()+"')",
+				"databaseUser.username", "database.name",
+			)
+		</div>
+	</div>
+}
+
+func getDatabasesTabHeaderItems(
+	selectedDatabaseType valueObject.DatabaseType,
+) []componentStructural.TabHeaderItem {
+	tabHeaderItems := []componentStructural.TabHeaderItem{
+		{Label: "MySQL/MariaDB/Percona", ComponentValue: "mariadb"},
+		{Label: "PostgreSQL", ComponentValue: "postgresql"},
+	}
+
+	for itemIndex := range tabHeaderItems {
+		if selectedDatabaseType.String() != tabHeaderItems[itemIndex].ComponentValue {
+			tabHeaderItems[itemIndex].AdditionalClasses = " text-slate-500"
+			continue
+		}
+
+		tabHeaderItems[itemIndex].AdditionalClasses = " text-speedia-500 border-b border-speedia-500"
+	}
+
+	return tabHeaderItems
+}
+
+templ DatabasesTabs(selectedDbTypeSummary presenterDto.SelectedDatabaseTypeSummary) {
+	@componentStructural.TabHeader(
+		getDatabasesTabHeaderItems(selectedDbTypeSummary.Type), "changeSelectedDatabaseType",
+	)
+	<!-- DatabaseTabContent -->
+	<div class="bg-os-800 p-4 rounded-b-xl rounded-r-xl">
+		<div class="px-4 bg-os-500 rounded-lg space-y-3">
+			if !selectedDbTypeSummary.IsInstalled {
+				<div class="p-4 bg-os-500 rounded-r-lg">
+					<i class="ph-duotone ph-warning align-middle text-xl text-yellow mr-3"></i>
+					<span class="mr-3">The<p class="inline-flex mx-1 text-speedia-300 font-bold">{ selectedDbTypeSummary.Type.String() }</p>service is not installed yet.</span>
+					@componentForm.SubmitButton("Schedule "+selectedDbTypeSummary.Type.String()+" service installation", "ph-caret-line-down", "installDatabaseService('"+selectedDbTypeSummary.Type.String()+"')", true)
+				</div>
```

Comment:
I would put this inside its own templ and use it on the else. The rule / common pattern is that someone accessing the page already installed the service, so you must assume the Install service part as an exception.

Reply:
I found it quite confusing to decide how this should be done because I was treating the non-installed service as an exception. Since `if` statements are typically used for exceptions in our conventional code, I wanted to maintain the same approach here in Templ. Because we are required to use `else`, I had it handle the standard case while `if` continued handling the exception.

However, I have reversed the order and now have the `if` handling the standard case.